core/icons/comment_icon.ts

```diff
@@ -237,13 +237,24 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
     return this.bubbleVisiblity;
   }
 
-  setBubbleVisible(visible: boolean): void {
-    if (visible && (this.textBubble || this.textInputBubble)) return;
-    if (!visible && !(this.textBubble || this.textInputBubble)) return;
+  /**
+   * Sets whether the bubble is visible or not.
+   *
+   * @returns A promise that resolves when the the bubble has finished opening.
```

Comment:
This seems kind of misleading since you're not actually waiting on anything to finish. Seems like the `show*EditableBubble` methods would have to also return promises in order for this description to be accurate. Currently not sure of the value of adding these immediately-resolving promises. Is this the same thing you're proposing to do in the flyout.show method?

Reply:
I removed these in the previous PR, and I'm not planning to add promises to the flyout related stuff (based on your feedback)!